src/data/faq_de.json

```diff
@@ -158,11 +158,23 @@
+                            "Wir haben jedoch festgestellt, dass dieses Ergebnis fehlinterpretiert werden kann, z.&nbsp;B. wenn eine unvollständige Impfung überprüft wurde, was zu der falschen Schlussfolgerung führen konnte, dass für dieses Zertifikat keine Einreisebeschränkungen bestehen. Dies haben wir mit der Version 2.8 klargestellt: Für den Fall, dass ein Mitgliedsland keine Einreiseregeln in einem maschinenlesbaren Format für die Corona-Warn-App zur Verfügung gestellt hat, lautet die Überschrift nun ‚Zertifikat nicht prüfbar', während die folgenden Texte gleich geblieben sind. Dies bedeutet <b>nicht</b>, dass Ihr Zertifikat ungültig ist oder dass Sie nicht in das Land einreisen dürfen, für das Sie Ihr Zertifikat überprüft haben. In diesem Fall überprüfen Sie bitte Ihren Status anhand der Informationen, die Sie unter <a href='https://reopen.europa.eu/de' target='_blank' rel='noopener noreferrer'>https://reopen.europa.eu/de</a> finden. Weitere Informationen zu dieser Änderung finden Sie im zweiten Absatz dieses Blog Posts: <a href='../blog/2021-08-25-cwa-version-2-8/' target='_blank' rel='noopener noreferrer'>Mit Version 2.8 der Corona-Warn-App werden Namen in Zertifikaten in standardisierter Schreibweise angezeigt</a>.<hr>",
+                            "<b>Warum zeigt die CovPass-App ein anderes Ergebnis an, wenn ich die Einreisregeln für ein Land prüfe, für das noch keine Einreiseregeln in einem maschinenlesbaren Format vorliegen?</b>",
+                            "Die Gültigkeitsprüfung für Einreiseregeln wurde erstmals in der Corona-Warn-App mit Version 2.6 eingeführt. Als diese Gültigkeitsprüfung für CovPass entwickelt wurde, diente der Ergebnisbildschirm der Corona-Warn-App als Vorlage für die aktuelle Implementierung von CovPass, und beide Apps zeigten Ergebnisse für Länder ohne Einreiseregeln in einem maschinenlesbaren Format in gleicher Weise an. Nach dem Update auf die Version 2.8 zeigt die Corona-Warn-App diese Ergebnisse nun in einer verbesserten Weise an, während die Anzeige in der CovPass-App unverändert blieb. Wir arbeiten an einer gemeinsamen Lösung für beide Apps."

```

Comment:
For the moment remove this, please. Once aligned with the other stakeholders we should reintroduce this. Thanks.